sdktests/common_tests_eval.go

```diff
@@ -77,6 +76,12 @@ func (c CommonEvalParameterizedTestRunner[SDKDataType]) runTestSuite(
 	}
 	client := NewSDKClient(t, append(clientConfig, dataSource)...)
 
+	// We can't rely on the test framework's usual auto-closing of the client, because this method could
+	// be called multiple times for a single value of t.
+	defer func() {
+		_ = client.Close()
+	}()
```

Comment:
Neglecting to close an SDK client before starting another SDK client is not a problem for server-side SDKs, and it's not a problem with node-client-sdk which I had been testing with earlier, but it does cause a failure in SDKs like dotnet-client-sdk that enforce singleton usage. It's not an issue in most of the tests because any client created within the scope of a `t *ldtest.T` gets automatically closed at the end of that test or subtest— but in this particular case, this helper method wasn't always associated with a single test scope.